(Week 5) Visual evaluation of the effects of quantization

voyagerx3 opened the pull request.

vicente-gonzalez-ruiz commented:
Thanks @igj281 

Please, could you also see if the docstrings I have written in the file `src/MCDWT.py` is processed by Sphinx?

Please, remember to create a new issue for that before start working on in.

Tons of thank!
Vi.

voyagerx3 committed:
inicio

voyagerx3 committed:
add   src/test_sum_sub.sh

voyagerx3 committed:
numerar desde 001->001

voyagerx3 committed:
creacion del directorio dwt

voyagerx3 committed:
actualizacion

Commit:
Merge branch 'master' of https://github.com/voyagerx3/MCDWTold

voyagerx3 committed:
add gitignore

voyagerx3 committed:
Update issue02.py

Commit:
add Extraerimg

what happened

 Changes to be committed:
	new file:   src/extraerimg.sh
	modified:   src/issue02.py

voyagerx3 committed:
Update issue02.py

Commit:
actualizacion

Commit:
gitignore

Commit:
.

Commit:
try catch division 0

what happened

[ticket: X]

Commit:
MCDWT.py original

MCDWT_I3.py Version GCVG

split bhb.py bhc.py

[ticket: ISSUE 3]

Commit:
actualizacion

what happened

[ticket: X]

Commit:
conflict

Commit:
conflict

Commit:
conflict 02

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Actualizacion

what happened

[ticket: X]

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
issue5

what happened

[ticket: X]

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
src/MCDWT.py

what happened

[ticket: X]

Commit:
test issue05
what happened

[ticket: X]

Commit:
Norma

what happened

[ticket: X]

Commit:
div img función

what happened

[ticket: X]

Commit:
title for Images Factor

what happened

[ticket: X]

Commit:
Edicion de divimg

what happened

[ticket: X]

Commit:
Reconstruccion de la Imagem

what happened

[ticket: X]

Commit:
actualizacion del resultado

what happened

[ticket: X]

Commit:
actualizacion del Notebook

what happened

[ticket: X]
